tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -55,3 +55,97 @@ def test_ndarray(self, xp, sp):
         b = xp.array(self.b, dtype=self.dtype)
         x = sp.linalg.lsqr(A, b)
         return x[0]
+
+
+@testing.parameterize(*testing.product({
+    'ord': [None, -numpy.Inf, -2, -1, 0, 1, 2, 3, numpy.Inf, 'fro'],
+    'dtype': [
+        numpy.float32,
+        numpy.float64,
+        numpy.complex64,
+        numpy.complex128
+    ],
+}))
+@unittest.skipUnless(scipy_available, 'requires scipy')
+@testing.gpu
+class TestMatrixNorm(unittest.TestCase):
+
+    @testing.numpy_cupy_allclose(rtol=1e-3, atol=1e-4, sp_name='sp', 
+                                 type_check=False,
+                                 accept_error=(ValueError, 
```

Comment:
Could you remove trailing spaces?  (See the result of Travis CI.)
```suggestion
                                 accept_error=(ValueError,
```

Reply:
done